S3/axe-core-package-gen.sh

```diff
@@ -0,0 +1,26 @@
+tag=${git tag --format "%(refname:short) %(creatordate:short)" | tail -n1 | awk '{print$1}'};
+date=${git tag --format "%(refname:short) %(creatordate:short)" | tail -n1 | awk '{print$2}'};
+
+
+generate_version_files () {
+        mkdir $tag;
+	git checkout tags/$tag -b master; mkdir $tag;
```

Comment:
```suggestion
        mkdir $tag;
	git checkout tags/$tag -b master;
```

No need to do this twice.